packages/typespec-client-generator-core/README.md

```diff
@@ -565,7 +565,7 @@ you need to take care of all related models/enums.
 
 ##### Target
 
-`Model | Enum | Union`
+`Model | Enum | Union | Namespace`
```

Comment:
Could you also please add some documentation on what it means for usage to be applied to `Namespace`? For Model, Enum and Union it's straightforward but for Namespace it might not be obvious what this means. So, the table you had in the other issue might be worth adding here.